pkg/healthcheck/healthcheck_multicluster.go

```diff
@@ -152,6 +152,54 @@ func (hc *HealthChecker) multiClusterCategory() category {
 					return hc.checkDaisyChains()
 				},
 			},
+			{
+				description: "all mirrored services have endpoints",
+				hintAnchor:  "l5d-multicluster-services-endpoints",
+				warning:     true,
+				check: func(ctx context.Context) error {
+					return hc.checkIfMirroredServicesHaveEndpoints()
+				},
+			},
+			{
+				description: "all gateway mirrors have endpoints",
+				hintAnchor:  "l5d-multicluster-gateways-endpoints",
+				warning:     true,
+				check: func(ctx context.Context) error {
+					return hc.checkIfGatewayMirrorsHaveEndpoints()
+				},
+			},
+			{
+				description: "matching gateway mirrors of all mirrored services exist",
```

Comment:
This description was really hard for me to understand.  What about "all mirror services have a gateway mirror"?

Reply:
Hmmm, but that would not express that the gateway mirror actually exists? Mirror services can have a remote gateway name as annotations that does not exist :thinking:

Reply:
perhaps "all mirror services have a valid gateway mirror".  a gateway that doesn't exist wouldn't be considered "valid"